mixer/pkg/lang/externs.go

```diff
@@ -127,14 +127,16 @@ var ExternFunctionMetadata = []ast.FunctionMetadata{
 	},
 }
 
-func externIP(in string) ([]byte, error) {
+// ExternIP creates an IP address
+func ExternIP(in string) ([]byte, error) {
 	if ip := net.ParseIP(in); ip != nil {
 		return []byte(ip), nil
 	}
 	return []byte{}, fmt.Errorf("could not convert %s to IP_ADDRESS", in)
 }
 
-func externIPEqual(a []byte, b []byte) bool {
+// ExternIPEqual compares two IP addresses for equality
+func ExternIPEqual(a []byte, b []byte) bool {
```

Comment:
These are exposed because the outside runtime is going to call them I suppose.

Reply:
Yeap. There is no CEL here, just refactor to accommodate it.